make/photon/registry/Dockerfile

```diff
@@ -13,7 +13,6 @@ COPY entrypoint.sh /
 RUN chmod u+x /entrypoint.sh
 
 RUN mkdir -p /etc/registry
-COPY config.yml /etc/registry/config.yml
```

Comment:
Why remove?

Reply:
no use anymore as the config is generated by prepare.